compiler/Makefile

```diff
@@ -115,6 +115,14 @@ $(CHPL_CONFIG_CHECK): | $(CHPL_BIN_DIR)
 
 COMPILER_LIB_DIR = $(CHPL_MAKE_HOME)/lib/compiler/$(CHPL_MAKE_HOST_BIN_SUBDIR)
 CMAKE_FLAGS = -DCMAKE_C_COMPILER=$(CHPL_MAKE_HOST_CC) -DCMAKE_CXX_COMPILER=$(CHPL_MAKE_HOST_CXX) -DCMAKE_RUNTIME_OUTPUT_DIRECTORY=$(CHPL_BIN_DIR) -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=$(COMPILER_LIB_DIR) -DCHPL_CXX_FLAGS="$(COMP_CXXFLAGS)" -DCHPL_LD_FLAGS="$(LDFLAGS)"
+ifdef VERBOSE
+	CMAKE_FLAGS += -DCMAKE_VERBOSE_MAKEFILE=ON
+	CMAKE_FLAGS += -DCMAKE_MESSAGE_LOG_LEVEL=VERBOSE
+else
+	CMAKE_FLAGS += -DCMAKE_VERBOSE_MAKEFILE=OFF
```

Comment:
if the environment variable `CMAKE_VERBOSE_MAKEFILE` is set, but `VERBOSE` is not, as with `CMAKE_VERBOSE_MAKEFILE=1 make`, wouldn't this set `-DCMAKE_VERBOSE_MAKEFILE=OFF` ? I guess `if (DEFINED ENV{CMAKE_VERBOSE_MAKEFILE})` is making it ignore the command line argument and use the env var?

Generally speaking, for a build run by make or cmake, I would expect a command line argument to take priority over an environment variable.